### PR TITLE
Versionless Beta Features

### PR DESCRIPTION
**Pull request by rsherget**
change reverted #28644 
reintroducing that change #28673 

for issue #28470

### CODE REVIEW

**Comment by rsherget** [comment]
#build

**Comment by rsherget** [comment]
#build

File: dev/com.ibm.websphere.appserver.features/test/src/com/ibm/ws/feature/utils/VersionlessFeatureCreator.java
Comment (tbitonti) [inline dev/com.ibm.websphere.appserver.features/test/src/com/ibm/ws/feature/utils/VersionlessFeatureCreator.java:1]: No change is needed, but usually, this:
                if(existingVersions.contains(s)){
                    existingVersions.remove(s);
                }
Can be just:
                existingVersions.remove(s);
Or, if you need to know either the prior element, or if the element was removed:
                boolean didRemove = existingVersions.remove(s);
Doing 'contains' *and* 'remove' does a redundant hash.

**Comment by rsherget** [comment]
#build

**Review by tbitonti — APPROVED** [review]

**Comment by rsherget** [comment]
#libby

**Pull request merged**